src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/Logger.cs

```diff
@@ -20,7 +20,7 @@ namespace OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper;
 
 internal static class Logger
 {
-    private static readonly IOtelLogger Log = OtelLogging.GetLogger();
+    private static readonly IOtelLogger Log = OtelLogging.GetLogger("AspNetCoreBootstrapper");
```

Comment:
`nameof(BootstrapperHostingStartup)` ?

Reply:
@RassK how about `nameof(AspNetCoreBootstrapper)`?

This is the name of the namespace where the logger is used so I think the name is good.